File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printPyramid(int height);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    printPyramid(height);
}

void printPyramid(int height)
{
    int spacesCount = height - 1;
    int hashSymbolCount = 1;
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < spacesCount; j++)
        {
            printf(" ");
        }
        spacesCount--;
        for (int j = 0; j < hashSymbolCount; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < hashSymbolCount; j++)
        {
            printf("#");
        }
        hashSymbolCount++;
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


while True:
    height = int(get_int("Height: "))
    if height > 0 and height < 9:
        break

for number in range(1, height + 1):
    print(" " * (height - number), end="")
    print("#" * number, end="")
    print(" " * 2, end="")
    print("#" * number, end="")
    print()